### DISCUSSION TITLE
显示汉化成功，但是游戏内文本还是原文 （已解决）

### DISCUSSION THREAD

**Discussion by Unaticlay** [GpointChen/FFXIVChnTextPatch-GP#179] [Q&A] [answered]
你好，我下载了最新的6.11 release然后按照使用手册的步骤进行了汉化， 软件显示汉化成功，但是游戏内文本并没有被替换，能问一下这是为什么吗？是因为游戏现在版本是6.15不兼容还是我步骤哪里做错了

**Comment by Unaticlay** [comment, marked as answer]
把语言包换成简体csv，就汉化成功了，不是很清楚为什么

**Reply by GpointChen** [reply]
不是很確定你遇到了什麼問題，不過恭喜你解決了
